litex/soc/integration/export.py

```diff
@@ -428,6 +428,15 @@ def print_svd_register(csr, csr_address, description, length, svd):
             svd.append('            </interrupt>')
         svd.append('        </peripheral>')
     svd.append('    </peripherals>')
+    if len(soc.mem_regions) > 0:
+        svd.append('    <vendorExtensions>')
+        for name, region in soc.mem_regions.items():
```

Comment:
Maybe put all of the memory regions inside a `<memoryRegions>` tag, similar to how all the `<peripheral>` tags are nested inside of a big `<peripherals>` tag? That way if we need to add more vendor extensions later on, we have the ability to do so.